src/components/header.js

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+import { Chromicons } from './icons/chromicons';
+import { CheckCircle, Download, Flag, Lifeology } from '@lifeomic/chromicons';
+import * as allLinedChromicons from '@lifeomic/chromicons';
+import clsx from 'clsx';
+
+const HeroNavLink = ({ className, children, ...rootProps }) => {
+  return (
+    <a
+      className={clsx(
+        'flex items-center px-2 duration-300 ease-in-out transition-opacity hover:opacity-75 focus:outline-none focus-visible:shadow-dark-mode-outline focus-visible:underline',
```

Comment:
Since this is no longer on a dark background, I think it's safe to delete `focus-visible:shadow-dark-mode-outline` from here